### ISSUE TITLE
Linting warnings in some typescript files

### ISSUE THREAD

**Issue by tahamukhtar20** [opencv/cvat#7598]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://opencv.github.io/cvat/docs/)

### Steps to Reproduce

run lint and you will face some warnings
similarly when you push some changes you will observe that there are some warnings visible=>
![image](https://github.com/opencv/cvat/assets/91777330/4b3ed5c4-6879-45bc-8fdd-1cd8b150a37c)

### Expected Behavior

No linting warnings

### Possible Solution

adding the required return types to fix this issue

### Environment

```Markdown
- Ubuntu 23.04
- Chrome Browser
```

**Comment by tahamukhtar20** [comment]
@bsekachev What do you think about this one, can I work on this?

**Comment by bsekachev** [comment]
Hello,
Are you going to fix all typescript warnings in the product? 
Or just enable linting? The second option will not really work well for people who deliver new features because they will also need to fix existing errors and warnings.

**Comment by tahamukhtar20** [comment]
No @bsekachev, I think linting is already enabled, there are some typescript warnings in some files, I noticed these initially when I was working on my issue which was previously assigned to me, so the first option, fixing all typescript errors in the product.

**Comment by bsekachev** [comment]
Under linting I actually meant adding one more stage on git pre-commit (now it runs eslint on changed files, for example) and adding a new action on GitHub.

**Comment by tahamukhtar20** [comment]
Yes, so there are some typescript warnings right now, for example, there are some types which are not added in the code, the code runs fine without those types but, it throws some warnings, =>
![image](https://github.com/opencv/cvat/assets/91777330/3ec677ec-7d3b-46ae-b5ec-d816afa316ae)
in this image we just need to add a return type which would be void in this case to remove the warning. 
I'm looking to fix such errors and not add any new action on GitHub as that would cause problems for other contributors and their workflows.
So, @bsekachev should I work on this? Please let me know if this needs some more explanation.

**Comment by bsekachev** [comment]
Hello, 
You are welcome to contribute to fix these warnings.
